pdaltools/standardize_format.py

```diff
@@ -60,6 +61,12 @@ def parse_args():
         help="List of extra dims to keep in the output (default=[], use 'all' to keep all extra dims), "
         "extra_dims must be specified with their type (see pdal.writers.las documentation, eg 'dim1=double')",
     )
+    parser.add_argument(
+        "--rename_dims",
+        nargs="*",
+        action="append",
+        help="Rename dimensions in pairs: --rename_dims old_name new_name ...",
```

Comment:
précise un peu pour le cas où il y a 2 dimensions à renommer:

help="Rename dimensions in pairs: --rename_dims old_name_dim1 new_name_dim1 old_name_dim2 new_name_dim2 ...",